frontend/components/how-it-work/index.jsx

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import { GradientText } from '../shared/Heading';
+import PersonCard from '../person-card';
+
+const HowItWork = ({ title, paragraph, cardDetails, sideImage, steps }) => {
+  return (
+    <div className="text-center my-16">
+      <div className="container relative">
+        <div className="absolute left-0 top-0 lg:-ml-10 opacity-25 md:opacity-100">
+          <img src={sideImage.url} />
+        </div>
+        <div>
+          <GradientText text={title} className="text-xxlg uppercase font-bold" />
```

Comment:
Use Heading component as described in the code example component in heading component

Reply:
I used it a described .. this the gradient heading just take the title and classnl name and gradient color no other thing !!